pom.xml

```diff
@@ -301,11 +301,11 @@
         <inbound.auth.oauth.version>6.5.3</inbound.auth.oauth.version>
         <commons-collections.version>3.2.0.wso2v1</commons-collections.version>
         <carbon.kernel.version>4.6.2-m9</carbon.kernel.version>
-        <identity.framework.version>5.20.33</identity.framework.version>
+        <identity.framework.version>5.20.51-SNAPSHOT</identity.framework.version>
```

Comment:
remove snapshot versions when other component PRs get merged

Reply:
yes. need to bump charon and framework versions